spec/suites/map/MapSpec.js

```diff
@@ -1317,6 +1317,68 @@ describe("Map", () => {
 			expect(map.getSize().x).to.equal(600);
 			expect(map.latLngToContainerPoint(center).x).to.equal(300);
 		});
+
+		it("auto invalidateSize after container resize", (done) => {
+			map.setView([0, 0], 0);
+			const spy = sinon.spy();
+			map.on("resize", spy);
+
+			expect(spy.called).to.not.be.ok();
+
+			map.getContainer().style.width = '200px';
```

Comment:
@IvanSanchez always fired over css size change?

Reply:
@Falke-Design I'm half asleep and totally skimmed over that really important bit. Damn.